### PR TITLE
Add e2e test for internal static IP

### PR DESCRIPTION
**Pull request by spencerhance**
/assign @freehan

### CODE REVIEW

**Comment by spencerhance** [comment]
@freehan Can you take a look at this?

File: cmd/e2e-test/ilb_test.go [hunk 1]
```diff
@@ -140,6 +150,63 @@ func TestILB(t *testing.T) {
 	}
 }
 
+func TestILBStaticIP(t *testing.T) {
```
Comment (freehan) [inline cmd/e2e-test/ilb_test.go:153]: Will the name got included with the existing `ILB` key word? You might just want to have lower case in it

Comment (spencerhance) [reply cmd/e2e-test/ilb_test.go:153]: it shouldn't since we use "ILB$" as the keyword

**Comment by spencerhance** [comment]
@freehan Sounds good, I changed it to a transition test

File: cmd/e2e-test/ilb_test.go [hunk 2]
```diff
@@ -140,6 +150,81 @@ func TestILB(t *testing.T) {
...
+			if err != nil {
+				t.Fatalf("e2e.WaitForIngress(s, %q) = _, %v; want _, nil", ing.Name, err)
+			}
+			if len(ing.Status.LoadBalancer.Ingress) < 1 {
+				t.Fatalf("Ingress does not have an IP: %+v", ing.Status)
+			}
+
+			vip := ing.Status.LoadBalancer.Ingress[0].IP
+			params := &fuzz.GCLBForVIPParams{VIP: vip, Validators: fuzz.FeatureValidators(features.All), Region: Framework.Region, Network: Framework.Network}
+			gclb, err = fuzz.GCLBForVIP(context.Background(), Framework.Cloud, params)
```
[the remark this answers is not in the document]
Comment (freehan) [reply cmd/e2e-test/ilb_test.go:216]: is this related to the recent test failure?

**Comment by spencerhance** [comment]
@freehan I rebased the commit, can you re-lgtm?

**Pull request merged**